drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/metadata/ScenarioHeaderMetaDataTest.java

```diff
@@ -44,11 +44,13 @@ public void constructorFail() {
         new ScenarioHeaderMetaData("", "", "", factoryMock, true, true);
     }
 
-    @Test(expected = IllegalStateException.class)
```

Comment:
@jomarko @manstis  @ederign 
Please see my previous comment and restore the annotation/implementation.